### PR TITLE
feat: add cd notification commands

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
New commands:
`plural cd notifications sinks list`
`plural cd notifications sinks upsert NAME`

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/cd_notifications.go [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+func (p *Plural) handleListNotificationSinks(_ *cli.Context) error {
+	if err := p.InitConsoleClient(consoleToken, consoleURL); err != nil {
+		return err
+	}
+	result := make([]*consoleclient.NotificationSinkFragment, 0)
+	var after *string
+	var pageSize int64
+	pageSize = 100
```
Comment (michaeljguarino) [inline cmd/plural/cd_notifications.go:99]: Looks link the linter wants this to be something like `pageSize := int64(100)`

Comment (michaeljguarino) [reply cmd/plural/cd_notifications.go:99]: Perhaps we should make a reusable paginator in the polly lib though

Comment (zreigz) [reply cmd/plural/cd_notifications.go]: I fixed the linter. I will think about a reusable paginator in the polly lib

Comment (zreigz) [reply cmd/plural/cd_notifications.go]: @michaeljguarino I used polly lib

**Review by michaeljguarino — APPROVED** [review]
fix go mod issues but this is good to go

**Pull request merged**